src/api/api.ts

```diff
@@ -39,11 +39,20 @@ export const insertData = async (input: IInputFormData) => {
 	return true
 }
 
+function dateToKorean(date: Date): string {
+	const year = date.getFullYear()
+	const month = String(date.getMonth() + 1).padStart(2, '0')
+	const day = String(date.getDate()).padStart(2, '0')
+	return `${year}-${month}-${day}`
```

Comment:
```suggestion
	return date.toLocalString('sv').slice(0,10)
```
스웨덴 포맷형식을 사용하면 한국시간대 및 스웨덴 포맷 (0000-00-00) 으로 편리하게 바꿀 수 있습니다.